app/views/teams/_show_mobile.html.erb

```diff
@@ -29,8 +28,8 @@
         <%= render 'shared/components/button_default_medium', label: 'View full dashboard', icon_name: 'icon-arrow-right' %>
       </div>
     <% end %>
-
-    <div class="my-6 flex justify-start gap-6">
+   <div data-controller="teams-tab-change">
+    <div class="mt-2 mb-6 flex justify-start gap-6">
       <a href="javascript:void(0);" data-action="click->teams-tab-change#changeTab" data-tab="user" class="text-base font-normal active-border">User</a>
```

Comment:
Can use use target name here to relate this link to the actual tab ?

Reply:
target name used here